mail_notification_custom_subject/models/mail_message_custom_subject.py

```diff
@@ -0,0 +1,36 @@
+# Copyright 2020 Tecnativa - João Marques
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+
+
+class MailMessageCustomSubject(models.Model):
+    _name = "mail.message.custom.subject"
+    _description = "Mail Message Custom Subject"
+
+    name = fields.Char(string="Template Name")
+    model_id = fields.Many2one(
+        comodel_name="ir.model",
+        string="Model",
+        required=True,
+        help="Model where this template applies",
+    )
+    subtype_ids = fields.Many2many(
+        comodel_name="mail.message.subtype", string="Applied Subtypes", required=True,
+    )
+    subject_template = fields.Char(
+        string="Subject Template",
+        required=True,
```

Comment:
It seems logical to me that this field should be translatable:

```suggestion
        required=True,
        translate=True,
```

Also, usually you'll need to define the lang as a template, to be able to use expressions such as `${object.partner_id.lang or user.lang}"

Reply:
This is intended for putting an expression, so making that expression translatable will over-complicate things. We don't have such use case, so I would keep it as is.

Reply:
> We don't have such use case

🤔 I think I don't understand what you mean. You don't have somebody whose native language isn't English?

I mean: I know people that systematically deletes mails with English subjects because they think they're spam.

We can leave this untranslatable if you prefer, but I feel like sooner or later we'll have to add translations.

Reply:
Yes, we are leaving this for now. We don't have multi-language users. It can go to the ROADMAP.